### PR TITLE
Bug 1870647 - Start the Nightly 123 development cycle

### PR DESCRIPTION
**Pull request by sarah541**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1870647

### CODE REVIEW

**Comment by t-p-white** [comment]
Just looking at the failing task, looks like `mozac_feature_addons_authors` is still being referenced somewhere?

**Comment by rahulsainani** [comment]
Seems like a string is being used in [layouts](https://searchfox.org/mozilla-mobile/search?q=%40string%2Fmozac_feature_addons_authors&path=&case=false&regexp=false) Not sure if we should remove it or simply update the layout

**Comment by sarah541** [comment]
> Seems like a string is being used in [layouts](https://searchfox.org/mozilla-mobile/search?q=%40string%2Fmozac_feature_addons_authors&path=&case=false&regexp=false) Not sure if we should remove it or simply update the layout

I think the intent was to remove that string and use this instead `mozac_feature_addons_author`.
According to this [PR](https://github.com/mozilla-mobile/firefox-android/pull/3955),

> The biggest change here is that we no longer have a list of authors for an add-on, we only have one author (optional). This is because Gecko only considers a single author.

**Comment by rocketsroger** [comment]
Looks like sample browser and reference browser still use that string.  We either remove the "moz:removedIn" since it's not a unused resource.  Or we replace them with `mozac_feature_addons_author` so we can remove this string.

**Comment by sarah541** [comment]
I've pushed the mozRemovedIn to "121" and filed a follow up [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1870668) so we can continue with this PR

**Review by rocketsroger — APPROVED** [review]

**Review by rahulsainani — APPROVED** [review]
🔢

File: android-components/components/feature/addons/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@
     <!-- The author of an add-on. -->
     <string name="mozac_feature_addons_author">Author</string>
     <!-- The authors of an add-on. -->
-    <string name="mozac_feature_addons_authors" moz:removedIn="120" tools:ignore="UnusedResources">Authors</string>
+    <string name="mozac_feature_addons_authors" moz:removedIn="121" tools:ignore="UnusedResources">Authors</string>
```
Comment (gabrielluong) [inline android-components/components/feature/addons/src/main/res/values/strings.xml:81]:
```suggestion
    <string name="mozac_feature_addons_authors" moz:removedIn="122" tools:ignore="UnusedResources">Authors</string>
```

**Pull request merged**